accelerator/real_accelerator.py

```diff
@@ -34,14 +36,18 @@ def _validate_accelerator(accel_obj):
     # accelerator.abstractor_accelerator
     # or deepspeed.accelerator.abstract_accelerator, consider accel_obj
     # is a conforming object
-    if not ((dsa1 != None and isinstance(accel_obj, dsa1)) or (dsa2 != None and isinstance(accel_obj, dsa2))):
+    if not ((dsa1 is not None and isinstance(accel_obj, dsa1)) or (dsa2 is not None and isinstance(accel_obj, dsa2))):
```

Comment:
Why is this change needed? How is it different from previous code?

Reply:
So, I found justification [here](https://peps.python.org/pep-0008/#programming-recommendations). Thanks for this code improvement!.

Reply:
Happy to!.